.build_image_manifest

```diff
@@ -1,4 +1,4 @@
 100644 blob 47cb6bf90db7b43772d0a77f4082d8552d0724fc	Dockerfile
 100644 blob b46311e84b12ee620fa133b0e48e53ec80bff479	go.mod
-100644 blob d9cf89fd4955c2c5f618eeaf9073498ea8a0997f	go.sum
+100644 blob b83728b6ed5004fc1ea456b6771e9d997ef00086	go.sum
```

Comment:
What was the change in `go.sum`? Was it a clean-up?

Reply:
:woman_shrugging: I don't know. I think it was done by the command I ran that updates the image(?)

Reply:
The command to update the image updates this file,  https://github.com/weaveworks/eksctl/commit/ad42b2044e3f4df8d374e9bd1f2d179830f5e212 was the culprit.

Reply:
So what should have happen is that after `go.sum` change the image should had been rebuilt and pushed. The GHA CI job is currently indicating exactly this - image is not up to date, albeit perhaps it's not spelling it out to clearly right now ;)